docs/user/extract-images.md

```diff
@@ -1,8 +1,5 @@
 # Extract Images
 
-> Please note: In order to use the following code you need to install optional
```

Comment:
Why remove this? This is still relevant as *Pillow* is an optional dependency.

Reply:
The example does not use Pillow, but some of the other examples do.

Reply:
But AFAIK image extraction will throw an *ImportError* when *Pillow* is not available.

Reply:
Reinstated. The file should be the same as what is in the current HEAD.